misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/command/BindingsGenerator.java

```diff
@@ -141,16 +146,36 @@ void generateJavaBindings() throws BindgenException {
 
     private void resolvePlatformLibraries() throws BindgenException {
         if (projectRoot != null) {
-            Manifest manifest = TomlParserUtils.getManifest(projectRoot);
-            if (manifest != null) {
-                List<Library> platformLibraries = manifest.getPlatform().getLibraries();
-                if (platformLibraries != null) {
-                    for (Library library : platformLibraries) {
-                        if (library.path != null) {
-                            classPaths.add(Paths.get(projectRoot.toString(), library.path).toString());
-                        } else if (library.groupId != null && library.artifactId != null && library.version != null) {
-                            new BindgenMvnResolver(outStream).mavenResolver(library.groupId, library.artifactId,
-                                    library.version, projectRoot, false);
+            File tomlFile = new File(Paths.get(projectRoot.toString(), BALLERINA_TOML).toString());
```

Comment:
Here you can get the toml from the project without separately loading it project.currentPackage().ballerinaToml()

Reply:
Fixed